cmd/osbuild-worker-executor/handler_build.go

```diff
@@ -84,11 +84,12 @@ func runOsbuild(buildDir string, control *controlJSON, output io.Writer) (string
 	// #nosec G204
```

Comment:
Maybe a small comment here like: `// the result is put into a tar because we get sparse file support for free this way` (and/or if there is more maybe also list that?).